src/FondOfSpryker/Yves/GoogleTagManager/Plugin/VariableBuilder/QuoteVariables/TransactionFieldProductsSkuPlugin.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace FondOfSpryker\Yves\GoogleTagManager\Plugin\VariableBuilder\QuoteVariables;
+
+use Exception;
+use FondOfSpryker\Yves\GoogleTagManager\Dependency\VariableBuilder\QuoteFieldPluginInterface;
+use Generated\Shared\Transfer\GooleTagManagerQuoteTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Shared\Log\LoggerTrait;
+use Spryker\Yves\Kernel\AbstractPlugin;
+
+/**
+ * @method \FondOfSpryker\Yves\GoogleTagManager\GoogleTagManagerFactory getFactory()
+ */
+class TransactionFieldProductsSkuPlugin extends AbstractPlugin implements QuoteFieldPluginInterface
+{
+    use LoggerTrait;
+
+    /**
+     * @param \Generated\Shared\Transfer\GooleTagManagerQuoteTransfer $gooleTagManagerQuoteTransfer
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param array $params
+     *
+     * @return \Generated\Shared\Transfer\GooleTagManagerQuoteTransfer
+     */
+    public function handle(
+        GooleTagManagerQuoteTransfer $gooleTagManagerQuoteTransfer,
+        QuoteTransfer $quoteTransfer,
+        array $params = []
+    ): GooleTagManagerQuoteTransfer {
+        try {
```

Comment:
Bin immer noch der Ansicht das das try and catch sinnvoller in der caller class ist

Reply:
Muss Julian hier recht geben, hab auch nochmal drüber nachgedacht und ist schon cooler wenn das Paket was Plugins ausführt die Fehler abfängt.

Man kann sonst nie erwartet das jemand alle exceptions in seinen Plugins catcht und im Zweifel haut's die ganze App weg.